changelogs/fragments/67013-purefb_fs_deprecate_nfs_parameter.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
```

```suggestion
removed_features:
```
(https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment)

You might also want to mention this in https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/porting_guides/porting_guide_2.10.rst#noteworthy-module-changes